Description: Add MODULE_VERSION to the vboxvideo driver, so DKMS can find it.
Author: Adam Conrad <adconrad@ubuntu.com>

--- virtualbox-5.0.18-dfsg.orig/src/VBox/Additions/linux/drm/vbox_drv.c
+++ virtualbox-5.0.18-dfsg/src/VBox/Additions/linux/drm/vbox_drv.c
@@ -55,6 +55,7 @@
 
 #include <drm/drmP.h>
 #include <drm/drm_crtc_helper.h>
+#include "version-generated.h"
 
 int vbox_modeset = -1;
 
@@ -293,4 +294,7 @@ module_exit(vbox_exit);
 MODULE_AUTHOR(DRIVER_AUTHOR);
 MODULE_DESCRIPTION(DRIVER_DESC);
 MODULE_LICENSE("GPL and additional rights");
+#ifdef MODULE_VERSION
+MODULE_VERSION(VBOX_VERSION_STRING);
+#endif
 
